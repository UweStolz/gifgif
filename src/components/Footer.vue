<template functional>
  <v-footer
    color="red darken-3"
    app
    padless
  >
    <v-col
      class="text-center"
      cols="12"
    >
      <div
        style="color: white;"
      >
        {{ new Date().getFullYear() }} - <strong>GIFGIF</strong>
      </div>
    </v-col>
  </v-footer>
</template>

<script lang="ts">
import Vue from 'vue';

export default Vue.extend({
  name: 'Footer',
  functional: true,
});
</script>
