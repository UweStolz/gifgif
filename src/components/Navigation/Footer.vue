<template>
  <v-footer
    color="red darken-3"
    app
    fixed
  >
    <v-col
      class="text-center"
      cols="12"
    >
      <div style="color: white;">
        {{ new Date().getFullYear() }} -
        <v-btn
          text
          tile
          color="white"
          :to="'/about'"
        >
          About
        </v-btn>
        - <strong>GIFGIF</strong>
      </div>
    </v-col>
  </v-footer>
</template>

<script lang="ts">
import { Vue, Component } from 'vue-property-decorator';

@Component
export default class Footer extends Vue { }
</script>
