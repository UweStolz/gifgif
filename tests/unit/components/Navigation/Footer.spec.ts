import Footer from '@/components/Navigation/Footer.vue';
import shallow from '../../../helper';

describe('Footer.vue', () => {
  it('renders properly', () => {
    const wrapper = shallow(Footer);
    expect(wrapper.element).toMatchSnapshot();
  });
});
