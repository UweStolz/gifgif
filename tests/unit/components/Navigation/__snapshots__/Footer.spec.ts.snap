// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.vue renders properly 1`] = `
<v-footer-stub
  app="true"
  color="red darken-3"
  height="auto"
  tag="footer"
  tile="true"
>
  <v-col-stub
    class="text-center"
    cols="12"
    tag="div"
  >
    <div
      style="color: white;"
    >
      
      2020 -
      
      <v-btn-stub
        activeclass=""
        color="white"
        tag="button"
        text="true"
        tile="true"
        to="/about"
        type="button"
      >
        
        About
      
      </v-btn-stub>
      
      - 
      <strong>
        GIFGIF
      </strong>
    </div>
  </v-col-stub>
</v-footer-stub>
`;
